packages/wallet/backend/src/rafiki/validation.ts

```diff
@@ -1,15 +1,6 @@
 import { z } from 'zod'
 import { EventType, PaymentType } from './service'
 
-export const ratesSchema = z.object({
```

Comment:
rate endpoint is used by rafiki and not wallet frontend, it should not be moved in shared

Reply:
removed from here and use this definition from rates/validation